dependencies.gradle

```diff
@@ -131,7 +131,7 @@ libraries.splunkObservability = "com.splunk:splunk-otel-javaagent:1.4.0"
 // gradle plugins
 libraries.asciidoctorGradlePlugin = "org.asciidoctor:asciidoctor-gradle-plugin:1.6.1"
 libraries.cargoGradlePlugin = "com.bmuschko:gradle-cargo-plugin:2.7.1"
-libraries.cargoDockerPlugin = "com.bmuschko:gradle-docker-plugin:3.0.8"
+libraries.cargoDockerPlugin = "com.bmuschko:gradle-docker-plugin:3.2.5"
```

Comment:
It would be good to know if something was failing in the build without this upgrade. So a line in the commit message about why this is needed would be helpful.

Reply:
Updated commit message.